dev/reloadable-server/src/main/scala/play/core/server/LagomReloadableDevServerStart.scala

```diff
@@ -58,7 +58,7 @@ object LagomReloadableDevServerStart {
     buildLink: BuildLink,
     httpPort:  Int
   ): ReloadableServer = {
-    mainDev(buildLink, Some(httpPort), None, "0.0.0.0")
+    mainDev(buildLink, Some(httpPort), None, "127.0.0.1")
```

Comment:
So the address that the services bind to is not configurable?

I honestly don't know how important that is, but I'm afraid that this could end up somehow breaking things for someone who it worked for before.

Reply:
Interestingly enough, the (currently unused) `mainDevOnlyHttpsMode` method above already takes an address parameter.

This code was all copied from Play, which does have the `httpAddress` parameter here:

https://github.com/playframework/playframework/blob/c038bc147b03285a1de7acc50037e1e7f438bedf/framework/src/play-server/src/main/scala/play/core/server/DevServerStart.scala#L48

It's passed through from `Reloader`:

https://github.com/playframework/playframework/blob/c038bc147b03285a1de7acc50037e1e7f438bedf/framework/src/run-support/src/main/scala/play/runsupport/Reloader.scala#L236

We could do the same in Lagom's `Reloader`.

Tracing backwards through callers, things diverge a bit from there. In Play, the address and port ultimately originate from these two sbt settings: `playDefaultPort` and `playDefaultAddress`:

https://github.com/playframework/playframework/blob/8ef9e47e1d0d877730a57d6b7d4312fe925bff92/framework/src/sbt-plugin/src/main/scala/play/sbt/PlaySettings.scala#L143-L144

In Lagom, the equivalent of `playDefaultPort` is called `lagomServicePort`:

https://github.com/lagom/lagom/blob/08b75d4fc294636bb4b735ea832073f07ef5137a/dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala#L315

So, I think there should be a `lagomServiceAddress` that is used to pass through to `mainDev` here by way of `Reloader.startDevMode` and `Reloader.startNoReload`. Plus equivalent for Maven.

Reply:
For consistency, I would use "address" instead of "host" in all of the new settings.